### ISSUE TITLE
New rule: Split packages into separate lines and sort them

### ISSUE THREAD

**Issue by setpill** [hadolint/hadolint#445]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

When you install a list of packages and they are not split into a single package per line and sorted alphabetically, hadolint swats you with a newspaper.

### Actual behavior

No swatting with newspapers observed.

### Steps to reproduce the behavior

Have the following line in your Dockerfile:

```Dockerfile
RUN apt-get update && apt-get install --no-install-recommends -y curl unzip libcurl4-openssl-dev libssl-dev libnghttp2-dev zlib1g-dev patchelf && \
  rm -rf /var/lib/apt/lists/*
```

According to the [best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments) this isn't right. But hadolint does not bat an eye.
